ank/src/cli_commands/wait_list.rs

```diff
@@ -115,3 +127,214 @@ impl<T: WaitListDisplayTrait> WaitList<T> {
         self.added_workloads.is_empty() && self.deleted_workloads.is_empty()
     }
 }
+
+//////////////////////////////////////////////////////////////////////////////
+//                 ########  #######    #########  #########                //
+//                    ##     ##        ##             ##                    //
+//                    ##     #####     #########      ##                    //
+//                    ##     ##                ##     ##                    //
+//                    ##     #######   #########      ##                    //
+//////////////////////////////////////////////////////////////////////////////
+
+#[cfg(any(feature = "test_utils", test))]
+fn generate_test_wait_list(
```

Comment:
make it public to share the test functionality  within the 'test_utils' context, or remove the 'feature' config item if it is not intended to be public
```suggestion
#[cfg(any(feature = "test_utils", test))]
pub fn generate_test_wait_list(
```

Reply:
Goo catch 👍, you are right, either public or not exposed to the outside ```suggestion
#[cfg(test)]
fn generate_test_wait_list(
```